packages/app-content-pages/next.config.js

```diff
@@ -44,12 +36,12 @@ const nextConfig = {
   env: {
     COMMIT_ID,
     PANOPTES_ENV,
-    SENTRY_CONTENT_DSN,
     APP_ENV
   },
 
   experimental: {
-    forceSwcTransforms: true
+    forceSwcTransforms: true,
+    optimizePackageImports: ['@zooniverse/react-components', 'grommet', 'grommet-icons']
```

Comment:
A similar change was made in app-project: https://github.com/zooniverse/front-end-monorepo/pull/5521#discussion_r1414636881